### PR TITLE
Use generated raid plug category hashes.

### PR DESCRIPTION
**Pull request by ryan-rushton**

### CODE REVIEW

File: src/app/loadout/known-values.ts [hunk 1]
```diff
@@ -22,12 +22,10 @@ export const slotSpecificPlugCategoryHashes = [
...
-// TODO generate this somehow so we dont need to maintain it
-export const raidPlugCategoryHashes = [
-  PlugCategoryHashes.EnhancementsSeasonOutlaw, // last wish
-  PlugCategoryHashes.EnhancementsRaidGarden, // garden of salvation
-  PlugCategoryHashes.EnhancementsRaidDescent, // deep stone crypt
-  PlugCategoryHashes.EnhancementsRaidV520, // vault of glass
-];
+export const raidPlugCategoryHashes: number[] =
+  Array.isArray(raidModPlugCategoryHashes) &&
+  raidModPlugCategoryHashes.every((pch) => typeof pch === 'number')
```
Comment (bhollis) [inline src/app/loadout/known-values.ts:27]: When is it not an array of numbers?

Comment (nev-r) [reply src/app/loadout/known-values.ts:27]: image
image

yeah don't think this verification is needed, as TS is already aware of this structure's types

Comment (ryan-rushton) [reply src/app/loadout/known-values.ts:27]: Yeah I am probably being overly cautious in the event something changes in an update. Feels weird importing json as an object. I will update it accordingly.

Comment (ryan-rushton) [reply src/app/loadout/known-values.ts]: Done

**Review by bhollis — APPROVED** [review]

**Pull request merged**